certified-connectors/Powell Teams/apiDefinition.swagger.json

```diff
@@ -0,0 +1,1142 @@
+{
+  "swagger": "2.0",
+  "info": {
+    "title": "Powell.Teams.Web.PublicAPI",
+    "description": "Powell Teams API endpoints documentation.",
+    "version": "V1"
+  },
+  "host": "apiteams.powell-software.com",
+  "basePath": "/",
+  "schemes": [
+    "https"
+  ],
+  "consumes": [],
+  "produces": [],
+  "paths": {
+    "/beta/$batch": {
+      "post": {
+        "tags": [
+          "beta"
+        ],
+        "summary": "Batch methods",
```

Comment:
@PowellSoftware Can you please provide a valid sentence for the summary value, that every one can understand.